CHANGELOG.md

```diff
@@ -4,6 +4,14 @@
 
 [1]: https://pypi.org/project/google-cloud-bigquery/#history
 
+### [1.26.1](https://www.github.com/googleapis/python-bigquery/compare/v1.26.0...v1.26.1) (2020-07-23)
+
+
+### Bug Fixes
+
+* address issues with concurrent BigQuery tests [([#3426](https://www.github.com/googleapis/python-bigquery/issues/3426))](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/3426) ([34ad9c6](https://www.github.com/googleapis/python-bigquery/commit/34ad9c619cfc6dcc5356e9375b8c76d41ddc16a1))
```

Comment:
This probably shouldn't be in the changelog or should be reworded? Since the actual change we made was moving the samples to this repo.

BTW, the link [#3426](https://github.com/googleapis/python-bigquery/issues/3426) results in 404.

Reply:
Yeah, the history merge is causing the issue here, it's actually an older python-docs-samples bug.  Will amend the changelog.